R/standardize_MAE.R

```diff
@@ -278,6 +278,44 @@ refine_rowdata <- function(rd, se, default_v = "Undefined") {
   rd
 }
 
+#' Set unique primary identifiers in the data.frame-like objects
+#'
+#' This function sets the primary field in the data.frame-like objects to be unique 
+#' by appending the secondary field in parentheses for duplicates.
+#'
+#' @param dt data.table, data.frame or DFrame with data
+#' @param primary_name string with the name of the primary field
+#' @param secondary_name string with the name of the secondary field
+#' @param sep string with separator added before suffix
+#' @return fixed input table with unique primary field in the table
+#' @examples
+#' col_data <- S4Vectors::DataFrame(CellLineName = c("ID1", "ID1"), clid = c("C1", "C2"))
+#' col_data <- set_unique_names_dt(col_data, primary_name = "CellLineName", secondary_name = "clid")
+#' @keywords standardize_MAE
+#' 
+set_unique_names_dt <- function(dt, primary_name, secondary_name, sep = " ") {
+  
+  stopifnot(any(
+    inherits(col_data, "data.table") ||
```

Comment:
```suggestion
    inherits(dt, "data.table") ||
```
There is no col_data variable define within this function